server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -621,11 +621,13 @@ public VMSnapshotResponse createVMSnapshotResponse(VMSnapshot vmSnapshot) {
         vmSnapshotResponse.setDisplayName(vmSnapshot.getDisplayName());
         UserVm vm = ApiDBUtils.findUserVmById(vmSnapshot.getVmId());
         if (vm != null) {
-            vmSnapshotResponse.setVirtualMachineid(vm.getUuid());
+            vmSnapshotResponse.setVirtualMachineId(vm.getUuid());
+            vmSnapshotResponse.setVirtualMachineName(vm.getHostName());
```

Comment:
I think display_name should be first choice, if it is null then we can go with host_name.

Reply:
okay @harikrishna-patnala

Reply:
Thanks @shwstppr, looks good to me now